src/components/inputs/CheckboxButton/CheckboxButton.js

```diff
@@ -1,5 +1,60 @@
-import styled from 'styled-components'
-import BaseCheckboxButton from './CheckboxButton.base'
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled, { css } from 'styled-components'
+import ErrorMessage from 'SRC/components/inputs/ErrorMessage'
+
+const BaseCheckboxButton = ({className, input, value, error, children, ...props}) => {
+  return (
+    <div className={className}>
+      <input id={value} type='checkbox' value={value} {...props} />
+      <label htmlFor={value}>
+        {children}
+      </label>
+      {error &&
+        <ErrorMessage>{error}</ErrorMessage>
+      }
+    </div>
+  )
+}
+
+const handleShapeStyles = shape => {
+  switch (shape) {
+    case 'square':
+      return css`
+        height: 12rem;
+        width: 12rem;
+      `
+    case 'rectangle':
+      return css`
+        height: 6.4rem;
+        width: 100%;
+      `
+    case 'circle':
+      return css`
+        width: 8rem;
+        height: 8rem;
+        border-radius: 50%;
+      `
+    default:
+      return css`
+        height: 3.2rem;
+        width: 6.4rem;
+      `
+  }
+}
+
+function borderColor (props) {
+  if (props.error) {
+    return props.theme.colors.flameOrange
+  }
+  return props.theme.colors.gray4
+}
+
+const checked = css`
+  color: ${props => props.theme.colors.white};
```

Comment:
Why do we need to have this CSS separate from the rest? It looks like we're not adding this conditionally, would it make sense to just inline it in line 89?